SPECS/perl-YAML-Tiny/perl-YAML-Tiny.spec

```diff
@@ -12,6 +12,12 @@ Distribution:   Mariner
 BuildArch:      noarch
 BuildRequires:  perl >= 5.28.0
 BuildRequires:  perl-generators
+BuildRequires:  perl(ExtUtils::MakeMaker)
```

Comment:
Why is `perl(ExtUtils::MakeMaker)` needed for non-test builds as well?

Reply:
From my observation, most of the `perl-*` packages have a `BR` dep on `perl(ExtUtils::MakeMaker)`.
I am not sure why/how they used to build without this dep.